pom.xml

```diff
@@ -1399,5 +1407,19 @@
         </pluginManagement>
       </build>
     </profile>
+    <profile>
```

Comment:
I don't follow that reasoning. Could you clarify?

I was referring to some technical-based justification that emphasizes the importance of these presets over other potential ones which could conceivably apply to all builders of the project. If you mean to suggest that Dima's individual use case has these particular preferences, that doesn't seem like a strong justification for me. We can't reasonably characterize everybody's presets into our POM, after all (unfortunate as that may be), and I would advise users with particular presets preferences to create individualized profiles in their settings.xml file, because that's what that's for.... that's how Maven is intended to work: pom.xml for universal settings, and settings.xml for individual preferences.